docs/dev/setup/jenkins-gitlab.rst.txt

```diff
@@ -1152,21 +1159,21 @@ Prerequisites:
       Jenkins SSH Credentials
 
 8. Add a new node (select a name and select `Permanent Agent`):
```

Comment:
@sleiss is the indentation for step 8 and 9 intentional? I think you added this part.

Reply:
Just talked to Simon, and it probably wasn't intentional.